loom/common/core/telemetry.cpp

```diff
@@ -17,17 +18,22 @@ int Telemetry::tickId = 0;
 TableValues<TickMetricValue> Telemetry::tickValues;
 
 loom_precision_timer_t Telemetry::tickTimer = loom_startTimer();
-utArray<utString> Telemetry::tickTimerStack;
-TableValues<TickMetricRange> Telemetry::tickRanges;
+
+int Telemetry::tickProfilerRootsVisited;
+size_t Telemetry::eventsStartPos;
+bool Telemetry::tickProfilerActive = false;
+int Telemetry::tickThreadId = -1;
 
 // Initialize specialized constants for every type used
 // TickMetricValue
 template<> const TableType TableValues<TickMetricValue>::type = 1;
 template<> const int TableValues<TickMetricValue>::packedItemSize = 4 + 8;
 // TickMetricRange
 template<> const TableType TableValues<TickMetricRange>::type = 2;
-template<> const int TableValues<TickMetricRange>::packedItemSize = 4 + 4 * 4 + 2 * 8;
+template<> const int TableValues<TickMetricRange>::packedItemSize = 4 + 4 * 4 + 3 * 8;
```

Comment:
Why this change? Can we generate this from constants or note what the magic #s are?